org.altaqwaa.Altaqwaa.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.altaqwaa.Altaqwaa
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+separate-locales: false
+rename-icon: org.altaqwaa.rn0x
+finish-args:
+  - --socket=wayland
```

Comment:
does it really support waylan out of the box?

Reply:
Looks like running it with ozone _does_ have some issues like missing tray icons. I'll remove it for now till I sort this with upstream.